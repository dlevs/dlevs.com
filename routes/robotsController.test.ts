'use strict';

const { testControllerSnapshots } = require('../tests/testLib/testUtils');
const robotsController = require('./robotsController')();

testControllerSnapshots(robotsController, {
	index: { contexts: [{}] },
});
