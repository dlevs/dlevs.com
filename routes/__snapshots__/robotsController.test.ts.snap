// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index data passed to template with ctx {} 1`] = `
Object {
  "ctx": Object {
    "body": "# Allow crawling of all content
User-agent: *
Disallow:

# XML Sitemaps
Sitemap: undefined/sitemap.xml",
    "query": Object {},
    "render": [Function],
    "state": Object {
      "CANONICAL_URL": "DUMMY_CANONICAL_URL",
    },
  },
  "renderedData": undefined,
}
`;
